### ISSUE TITLE
Info graphic showing next level

### ISSUE THREAD

**Issue by godbit** [Lapsang-boys/Rex-Temporis#81]
Create a gui-element, for instance below the scoreboard (in the same style) showing info about the upcoming level (composition, armor etc).

Basically the same as writing -next, the difference being that the information would be there constantly. This is a proposal and probably polish beyond alpha/beta-state.

**Comment by karlek** [comment]
I like the idea, and definitely think we should implement this. We just need to decide on the presentation as you mentioned.

How do we make it clear with many different units in a squad? How do we denote special levels (which more and more are)? How granular should be be, basic things such as health, armor and movement speed or just armor type and squad / special abilities?

**Comment by godbit** [comment]
Yeah we have to sit down together and decide this i think. We probably need to make a function that reads the levels-object (if i remeber correctly) and takes the squadchain and automatically generates the appropriate information for each level.
